src/vivarium/interface/cli.py

```diff
@@ -99,24 +101,21 @@ def test():
     provided in the examples directory.
     """
     logging.basicConfig(level=logging.DEBUG)
-    model_specification = Path(vivarium.__file__).resolve().parent / 'examples' / 'disease_model' / 'disease_model.yaml'
-    results_directory = Path('~/vivarium_results').expanduser()
+    model_specification = disease_model.get_model_specification_path()
 
     try:
-        run_simulation(str(model_specification), str(results_directory))
+        run_simulation(model_specification)
         click.echo()
         click.secho("Installation test successful!", fg='green')
     except (BdbQuit, KeyboardInterrupt):
         raise
     except Exception as e:
-        logging.exception("Uncaught exception {}".format(e))
+        _log.exception("Uncaught exception {}".format(e))
         raise
-    finally:
-        shutil.rmtree(results_directory, ignore_errors=True)
 
 
 @simulate.command()
-@click.argument('model_specification', callback=verify_yaml,
+@click.argument('model_specification',
```

Comment:
These callbacks are not useful.  They were wrappers around model specification validation, which is already happening when the sim is initialized.